Cargo.toml

```diff
@@ -3,6 +3,11 @@ name = "rustodon"
 version = "0.1.0"
 authors = ["The Rustodon team"]
 
+[features]
+default = ["postgres"]
+postgres = []
+sqlite = []
```

Comment:
I think this could be

```toml
postgres = ["diesel/postgres", "diesel_infer_schema/postgres"]
sqlite = ["diesel/sqlite", "diesel_infer_schema/sqlite"]
```

Then remove those features from diesel and diesel_infer_schema below, and diesel will be compiled with or without the database engines depending on how you want to compile rustodon.

Reply:
Thanks for the tip! I've applied that in 4ac431193dc05633e77b2da7099a4c5d8a341efc.